### PR TITLE
Improve layout of blog index page

### PR DESCRIPTION
**Pull request by mosabua**
- use highlight image properly
- show latest three with large display
- show nine more in three rows of three
- remove subheader

Preview at https://deploy-preview-345--trino.netlify.app/blog/

### CODE REVIEW

File: _posts/2022-09-12-tenth-birthday-celebration-recap.md [hunk 1]
```diff
@@ -3,14 +3,13 @@ layout: post
 title: "Trino's tenth birthday celebration recap"
 author: "Brian Olsen"
 excerpt_separator: <!--more-->
+image: /assets/blog/trino-tenth-birthday/creators.jpeg
 ---
 
 What an exciting month we had in August! August marked the ten-year birthday of
```
Comment (colebow) [inline _posts/2022-09-12-tenth-birthday-celebration-recap.md:9]: We need a little bit of whitespace or a div with a small margin or something before this line, it's currently awkwardly close to the image on the blog page itself with no gap.

image

**Review by colebow — APPROVED** [review]
Looks like a massive improvement, just the one change necessary.

File: _layouts/post.html [hunk 2]
```diff
@@ -21,6 +21,11 @@ <h1 class="post-title p-name" itemprop="name headline">
   </header>
 
   <div class="content width clearfix" itemprop="articleBody">
+    {%- if page.image -%}
+    <div>
```
Comment (colebow) [inline _layouts/post.html:25]: Oh, I think we need the margin here.

**Review by bitsondatadev — APPROVED** [review]
Agree with @colebow, that margin on the blog itself would help a lot. This definitely looks better. 

This did just make me think that when we do paginate, we should use jekyll-paginate-v2 gem because the v1 doesn't support any notion of "first page". https://github.com/sverrirs/jekyll-paginate-v2/issues/28#issuecomment-312725026

V2 now does and that issue fixes it and links to an example of how to use it. FYI Cole.

File: _layouts/post.html (hunk 2 above)
Comment (mosabua) [reply _layouts/post.html:25]: I just kept it clean by using br

**Review by martint — APPROVED** [review]

**Comment by mosabua** [comment]
Thank you @martint !

**Pull request merged**